lib/search.rb

```diff
@@ -254,7 +254,7 @@ def self.execute(term, opts = nil)
   def execute(readonly_mode: Discourse.readonly_mode?)
     if log_query?(readonly_mode)
       status, search_log_id = SearchLog.log(
-        term: @term,
+        term: @clean_term,
```

Comment:
`@term` contains the search term stripped of special keywords like `@user` or `in:title` and the like.